application-admintools-default/src/main/java/com/xwiki/admintools/internal/uploadJob/UploadJob.java

```diff
@@ -122,6 +127,7 @@ protected void runInternal()
             batchRestore();
         } finally {
             this.progressManager.popLevelProgress(this);
+            logger.debug("Finished upload job with ID: [{}]", this.status.getJobID());
```

Comment:
i think it would be better to have "info" level logs. The debug level is used for debugging purposes and can print more detailed information.